docs/standard/containerized-lifecycle-architecture/design-develop-containerized-apps/visual-studio-tools-for-docker.md

```diff
@@ -3,88 +3,87 @@ title: Visual Studio Tools for Docker on Windows
 description: Get to know the Docker tools available in Visual Studio 2017 version 15.7 and later.
 author: CESARDELATORRE
 ms.author: wiwagn
-ms.date: 11/23/2018
+ms.date: 02/15/2019
 ms.custom: vs-dotnet
 ---
-# Using Visual Studio Tools for Docker (Visual Studio on Windows)
+# Use Docker Tools in Visual Studio 2017 on Windows
 
-The Visual Studio Tools for Docker development workflow is similar to the workflow when using Visual Studio Code and Docker CLI. In fact, it's based on the same Docker CLI, but it's easier to get started, simplifies the process, and provides greater productivity for the build, run, and compose tasks. Execute and debug your containers via simple actions like **F5** and **Ctrl**+**F5**. With the optional container orchestration support, in addition to being able to run and debug a single container, you can run and debug a group of containers (a whole solution) at the same time.
-
-> [!NOTE]
-> This article applies to Visual Studio on Windows, and not Visual Studio for Mac.
+The developer workflow when using the Docker Tools included in Visual Studio 2017 version 15.7 and later, is similar to using Visual Studio Code and Docker CLI (in fact, it's based on the same Docker CLI), but it's easier to get started, simplifies the process, and provides greater productivity for the build, run, and compose tasks. It can also run and debug your containers via the usual `F5` and `Ctrl+F5` keys from Visual Studio. You can even debug a whole solution if its containers are defined in the same `docker-compose.yml` file at the solution level.
 
 ## Configure your local environment
 
-With the latest versions of Docker for Windows ([https://docs.docker.com/docker-for-windows/](https://docs.docker.com/docker-for-windows/)), the straightforward setup makes it easy to develop Docker applications.
+With the latest versions of Docker for Windows, it is easier than ever to develop Docker applications because the setup is straightforward, as explained in the following references.
 
-Docker support is included in Visual Studio 2017. Download Visual Studio 2017 here: [https://aka.ms/vsdownload?utm_source=mscom&utm_campaign=msdocs](https://aka.ms/vsdownload?utm_source=mscom&utm_campaign=msdocs)
+> [!INFORMATION]
```

Comment:
It's occurred to me that INFORMATION is not a supported note label, so it doesn't render very well. Better would be NOTE. That's an issue throughout the documentation, if you want to address it.

Reply:
Sure, I will.

@mairaw and me agreed that I'd do some global Regex changes to fix contractions when we finish with these review, so I'll include the !INFORMATION -> !NOTE update.